src/Arcus.WebApi.Security.Authentication/SharedAccessKey/SharedAccessKeyAuthenticationFilter.cs

```diff
@@ -20,21 +20,23 @@ namespace Arcus.WebApi.Security.Authentication.SharedAccessKey
     /// </remarks>
     public class SharedAccessKeyAuthenticationFilter : IAsyncAuthorizationFilter
     {
-        private readonly string _headerName, _secretName;
+        private readonly string _headerName, _queryParameterName, _secretName;
 
         /// <summary>
         /// Initializes a new instance of the <see cref="SharedAccessKeyAuthenticationFilter"/> class.
         /// </summary>
         /// <param name="headerName">The name of the request header which value must match the stored secret.</param>
+        /// <param name="queryParameterName"></param>
```

Comment:
Can you provide documentation here as well please?

Reply:
Will do